client/packages/host/src/Host.tsx

```diff
@@ -58,6 +59,16 @@ Bugsnag.start({
 const skipRequest = () =>
   LocalStorage.getItem('/auth/error') === AuthError.NoStoreAssigned;
 
+/**
+ * Empty component which can be used to call startup hooks.
+ * For example, this component is called when auth information such as user or store id changed.
+ */
+const Init = () => {
+  // Fetch pack units at startup. Note, the units are cached, i.e. they are not fetched repeatedly.
+  usePackUnits();
+  return <></>;
+};
```

Comment:
Thought about putting this into DashboardService because the dashboard is navigated when changing store or login in. Think this is a bit more obvious though...